openfl/utilities/fed_timer.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import os
+import sys
+import time
+from threading import Thread
+
+logger = logging.getLogger(__name__)
+
+class CustomThread(Thread):
+    def __init__(self, group=None, target=None, name=None,
+                    args=(), kwargs={}):
+        Thread.__init__(self, group, target, name, args, kwargs)
+        self._result = None
+        self._duration = None
+
+    def run(self):
+        if self._target is not None:
+            start = time.time()
+            self._result = self._target(*self._args, **self._kwargs)
+            self._exec_duration = round(time.time() - start, 3)
```

Comment:
There is a `_duration` field in `__init__`, should It be assigned here, or do I miss smth?

Reply:
oops! Thanks for pointing it out.
Its a typo in line 20.
updating in next commit.